src/app/faq-assistant/services/communication-faq-assistant.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
+import {Subject} from 'rxjs/Subject';
+
+@Injectable()
+export class CommunicationFaqAssistantService {
+  subject = new Subject<any>();
```

Comment:
This should be private property. And please, give more specific name.

Reply:
type void is not suitable for this property